accord-core/src/main/java/accord/coordinate/TxnExecute.java

```diff
@@ -74,8 +75,9 @@ private TxnExecute(Node node, TxnId txnId, Txn txn, FullRoute<?> route, Particip
     }
 
     @Override
-    protected void start(Set<Id> readSet)
+    protected void start(Iterable<Id> to)
     {
+        Set<Id> readSet = Sets.newHashSet(to);
```

Comment:
maybe switch to `IntHashSet` and accept it in `Commit`?

Reply:
ill make the change but I wonder if its not best to just do `List.contains`; I went with `Iterable` to make sure I didn't break anything... I feel like in practice the read set will be small as its 1 node per non-overlapping range.